doc/theoretical_description_binary_classification.rst

```diff
@@ -0,0 +1,92 @@
+.. title:: Theoretical Description : contents
+
+.. _theoretical_description_binay_classification:
+
+=======================
+Theoretical Description
+=======================
+
+For binary classification in the distribution-free setting, there are
+different ways of handling uncertainty quantification: calibration,
+probabilistic prediction (consisting in estimating confidence intervals or CIs) 
+and set prediction (consisting in estimating prediction sets or PSs). Gupta et al.
+have established a tripod of theorems that connect these three notions for
+score-based classifier [1]. We will use their notation to present these different concepts.
+
+In MAPIE, we focus specifically on set prediction for multi-class classification
+and on calibration for binary classification.
```

Comment:
The Top-Lable calibration is actually not really binary use-cases. Then, it would be better to use standard calibration techniques (as implemented on sklearn).

Reply:
Yes, in this case, I will adapt this paragraph.

Reply:
I make a suggestion in another comment.